### PR TITLE
XIVY-8512 Raise project build plugin version

### PR DESCRIPTION
**Pull request by alexsuter**
- Raise project build plugin version in all given repos
- Create PR on GitHub for each repo

### CODE REVIEW

File: build/raiseRepo.sh [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    cd "${cloneDir}"
+    git checkout -q -b "${newBranch}" "${sourceBranch}"
+
+    ${updateAction}
+
+    if [ "$dryRun" = "0" ]; then      
+      git push -u origin "${newBranch}"
+
+      gh auth login --with-token < ${tokenFile}
```
Comment (alexsuter) [inline build/raiseRepo.sh:36]: for GH CLI I need to work with a token ... maybe we can create a GitHub APP?

File: build/raise-build-plugin-version/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    string(name: 'branch', defaultValue: 'master', description: 'Source branch e.g. master, release/8.0')
+    string(name: 'newReleaseVersion', defaultValue: '10.0.0', description: 'New project-build-plugin release version e.g 9.3.0')
+    string(name: 'newSnapshotVersion', defaultValue: '10.0.0-SNAPSHOT', description: 'New project-build-plugin SNAPSHOT version e.g 9.3.1-SNAPSHOT')
+  }
+
+  stages {
+    stage('build') {
+      steps {
+        script {
+          sh "git config --global user.name 'ivy-team'"
```
Comment (alexsuter) [inline build/raise-build-plugin-version/Jenkinsfile:28]: which user and email?

Comment (ivy-lli) [reply build/raise-build-plugin-version/Jenkinsfile:28]: from parameters or also the ivy-team user is ok for me...

**Review by ivy-lli — APPROVED** [review]

File: build/raise-build-plugin-version/Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+          sh "git config --global user.email 'info@ivyteam.ch'"
+          withEnv(['GIT_SSH_COMMAND=ssh -o StrictHostKeyChecking=no']) {
+            withCredentials([file(credentialsId: 'github-ivyteam-token-repo-manager', variable: 'tokenFile')]) {
+              sshagent(credentials: ['github-axonivy']) {
```
Comment (alexsuter) [inline build/raise-build-plugin-version/Jenkinsfile:32]: this one is used, to checkout and to push the new commit

**Comment by alexsuter** [comment]
Keep current state. Try to improve this later.

**Pull request merged**